.github/workflows/tests.yml

```diff
@@ -0,0 +1,13 @@
+name: govuk-docker
+
+on: [push, pull_request]
```

Comment:
Do we just need `push` here? Otherwise, I think this will run the tests twice, every time someone pushes.

Reply:
```suggestion
on: push
```